src/components/form/widgets/code-editor.ts

```diff
@@ -24,6 +24,8 @@ export class CodeEditor extends React.Component {
                 value: value,
                 language: 'json',
                 lineNumbers: true,
+                fold: true,
+                lint: true,
```

Comment:
My initial reaction when realising this is in the form component was "These seem like good defaults, but should we really hardcode this here?". Then I realised we hardcode the language to be `json` 🙈

Unless it's somehow already possible to override these, we might want to add that possibility at some point. But until then, I don't see a problem hard-coding `fold` and `lint` to enabled 🤷‍♂️ 🙂